### ISSUE TITLE
Bug Report: Freezes during HaxeFlixel screen

### ISSUE THREAD

**Issue by realicraft** [FunkinCrew/Funkin#798]
### If you are playing the game in a browser, what site are you playing it from?
[Newgrounds](https://www.newgrounds.com/portal/view/770371) or [Itch.io](https://ninja-muffin24.itch.io/funkin)? Specify below.
Newgrounds
### If you are playing the game in a browser, what browser are you using?
Chrome (or a Chromium based browser, i.e. Edge, Opera, etc.), Firefox, or Safari? Specify blow.
Chrome
### If you are playing a downloaded version of the game, what operating system are you using?
Windows (`x86`), Windows (`x86_64`), Linux, or macOS? Specify below.

## What version of the game are you using? Look in the bottom left corner of the main menu.
Whatever's currently on Newgrounds (likely 2.8.0) (can't reach main menu, freezes before then)
## Have you identified any steps to reproduce the bug? If so, please describe them below. Use images if possible.
Launch the game and wait.
## Please describe your issue. Provide extensive detail and images if possible.
When trying to load the game, during the part where HaxeFlixel loads, it fails to get several mp4 files, which causes the game to freeze. Opening the console reveals 2 to 4 errors, which have similar messages, such as `[lime.utils.Preloader] ERROR: Error loading asset "assets/music/kickstarterTrailer.mp4"`. Files affected seem to include `kickstarterTrailer.mp4`, `ughCutscene.mp4`, `gunsCutscene.mp4`, and `stressCutscene.mp4`. The errors usually occur around 50-80%.

**Comment by MyChade** [comment]
The cutscene video files forcibly stop downloading at exactly 30 seconds, which is when HaxeFlixel/Lime reports "Error loading asset". It seems to be a timeout (hard-coded?), as the game loads normally if the videos load completely before 30 seconds.

tl;dr: Slow connections freeze if they take longer than 30 secs to load. See attached screenshot

<img width="540" alt="screenshot of various mp4 files that stopped downloading after 30 seconds" src="https://user-images.githubusercontent.com/8697807/115487846-5db02980-a20e-11eb-8729-bbc2f4380a39.png">

**Comment by danilaml** [comment]
I appear to experience similar/same issue in Chrome 90 (win 10 x64):
![image](https://user-images.githubusercontent.com/2046350/116828884-424c0380-abaa-11eb-8b56-a104beee24d2.png)

**Comment by foxsouns** [comment]
Oh, that's not good. This explains the freezes I was having earlier.
